temporal-opentracing/src/main/java/io/temporal/opentracing/internal/OpenTracingWorkflowInboundCallsInterceptor.java

```diff
@@ -82,4 +82,81 @@ public WorkflowOutput execute(WorkflowInput input) {
       workflowRunSpan.finish();
     }
   }
+
+  @Override
+  public void handleSignal(SignalInput input) {
+    Tracer tracer = options.getTracer();
+    SpanContext rootSpanContext =
+        contextAccessor.readSpanContextFromHeader(input.getHeader(), tracer);
+    Span workflowSignalSpan =
+        spanFactory
+            .createWorkflowSignalSpan(
+                tracer,
+                input.getSignalName(),
```

Comment:
I recommend, to match other SDKs, that workflow inbound signal, query, and update span names be `HandleSignal:<name>`, `HandleQuery:<name>`, and `HandleQuery:<name>` instead of reusing the same outbound names

Reply:
done